mlrun/runtimes/nuclio/application/application.py

```diff
@@ -377,6 +377,10 @@ def create_api_gateway(
             ),
         )
 
+        # force ssl redirect so that the application is only accessible via https
+        api_gateway.with_force_ssl_redirect()
```

Comment:
few questions
1. can we opt out?
2. can we enforce it for "production" / "enterprise" deployments?

For an instance, I would not require it for MLRun CE